test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -5216,6 +5228,51 @@ public void testModifiedFolder() throws Exception {
   }
 
   @Test
+  public void testModifiedFolderUpdatedDocumentsQuery() throws Exception {
+    String parentId = FOLDER.pad("FFF1");
+    String parentFolder = "/FFF1";
+    insertFolder(EPOCH_1970, parentId, parentFolder);
+    String folderId = FOLDER.pad("FFF2");
+    String folder = "/FFF1/FFF2";
+    insertFolder(FEB_1970, folderId, folder);
+    insertDocument(FEB_1970, DOCUMENT.pad("aaa"), folder + "/aaa", folderId);
+    insertDocument(MAR_1970, DOCUMENT.pad("bbb"), folder + "/bbb", folderId);
+
+    checkModifiedDocIdsPushed(
+        ImmutableMap.of(),
+        startPaths(folder),
+        new Checkpoint(FEB_1970, DOCUMENT.pad("aaa")),
+        makeExpectedDocIds(folder, folder, "bbb"),
+        new Checkpoint(MAR_1970, DOCUMENT.pad("bbb")));
+
+    StringBuilder query = new StringBuilder();
```

Comment:
There's no advantage to using StringBuilder instead of + here, and this is more verbose and less efficient.